chill-scala/src/main/scala/com/twitter/chill/ScalaKryoInstantiator.scala

```diff
@@ -207,26 +207,11 @@ class AllScalaRegistrar_0_9_2 extends IKryoRegistrar {
   }
 }
 
-/** Registrar for everything that was registered in chill 0.10.0 */
-class AllScalaRegistrar_0_10_0 extends IKryoRegistrar {
```

Comment:
This class was badly added by me on scala 2.13 migration ... should we keep it around and mark it as deprecated? or can we go ahead and just remove it since we don't expect users to be using it?